main.kt

```diff
@@ -0,0 +1,22 @@
+//this is a function that converts temperatures from Celsius to Fahrenheit and vice versa. it does the math of the conversion for the user and makes it so all the user has to do is input the temperature they want converted.
+
+fun main() { //the function is named main and has no parameters so it runs when the program starts.
+    val temperatures = list(0.0, 32.0, 100.0, -40.0, 75.0) //a list of temperatures in Celsius, they are doubles but you can use ints. I made this a Value/constant because it's immutable and shouldn't be changed while the function is running.
+
+    for (temp in temperatures) {  // this is a for loop, it runs through all the temperatures in the list and prints them out in the console. for each temperature in the list, the function is called and executed. it will never leave out a number unless the value is null.
+        val celsiusToFahrenheit = celsiusToFahrenheit(temp) //inside the loop, this line converts the current temp (which is assumed to be celsius) to fahrenheit using the celsiusToFahrenheit function. the result is stored in celsiusToFahrenheit variable.
+        val fahrenheitToCelsius = fahrenheitToCelsius(temp) //this line converts the same temperature "temp", but this time treating it as if it were in fahrenheit. the result is store in the fahrenheitToCelsius variable.
+        
+        println("$temp°C is ${"%.2f".format(celsiusToFahrenheit)}°F") //this line prints the current temperature in celsius and the converted temperature in fahrenheit. the %.2f is used to limit the number of decimal places to 2 so we don't get long doubles. realistically this could probably be shortened down to one.
```

Comment:
The way you are formatting your temps are different. the first $temp is one decimal place (0.1) and your second format is rounding to the second (0.10)